### ISSUE TITLE
Set service-worker.js to Cache-Control: max-age=0?

### ISSUE THREAD

**Issue by abdonrd** [Polymer/pwa-starter-kit#310]
Should we set `service-worker.js` to `Cache-Control: max-age=0`, right?

Something like this in the `server/app.yml`?

```yml
handlers:
- url: /service-worker\.js
  static_files: service-worker.js
  upload: service-worker\.js
  expiration: 0m
```

**Comment by keanulee** [comment]
Could be something added to prpl-server since that's the thing setting cache headers now and already does some special stuff for SW. See https://github.com/Polymer/prpl-server/blob/1e66a56002b294a8121a6089735690b7c45580f4/src/prpl.ts#L146

**Comment by keanulee** [comment]
Fixed by bumping to prpl-server@1.4.0 and deployed.

**Comment by abdonrd** [comment]
@keanulee awesome! Thanks Keanu! I deployed with `prpl-server@1.4.0`, and it works!
